pages/api/paybutton/download/transactions/[paybuttonId].ts

```diff
@@ -45,10 +45,13 @@ export default async (req: any, res: any): Promise<void> => {
     const timezone = userPreferredTimezone !== '' ? userPreferredTimezone : userReqTimezone
     let networkIdArray = Object.values(NETWORK_IDS)
     if (networkTicker !== undefined) {
-      const slug = Object.keys(NETWORK_TICKERS).find(key => NETWORK_TICKERS[key] === networkTicker)
-      const networkId = getNetworkIdFromSlug(slug ?? NETWORK_TICKERS.ecash)
+      const slug =  NETWORK_SLUGS_FROM_IDS[NETWORK_IDS[networkTicker]]
```

Comment:
Just noticed: why get the slug from the id if you're gonna get the id from the slug? if the goal is to get the id, just forget the slug, check if the ticker is valid (we have a variable with the valid network tickers) and then get the id with `NETWORK_IDS[networkTicker]`